onnxruntime/core/optimizer/identical_children_consolidation.cc

```diff
@@ -122,6 +122,7 @@ string_view IdenticalChildrenConsolidation::IdentityBuilder(const Graph& graph,
     }
     identity.append("####");
   }
-  return {identity};
+  string_view const identity_view{identity};
+  return identity_view;
```

Comment:
You are now returning a string_view of the string, that is going to be destroyed as the function returns.